index.js

```diff
@@ -25,6 +25,20 @@ module.exports.onRendererWindow = async (window) => {
   if (await isElevated()) {
     window.document.documentElement.classList.add('elevated');
   }
+
+  var config = window.config.getConfig();  
+  var deg = 0;
+
+  const updateBorderAngle = () => {
+    window.document.documentElement.style.setProperty('--border-angle', `${deg}deg`);
+    deg = deg > 360 ? 2 : deg + 2;
+  }
+
+  if (config.hyperBorder && config.hyperBorder.animate) {
+    console.log('start animation interval');
```

Comment:
Did you mean to leave the `console.log()` line?

Reply:
no. I will remove it.